services/ore/ore-stars.service.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+const { metric } = require('../text-formatters')
+const { BaseOreService, documentation, keywords } = require('./ore-base')
+
+module.exports = class OreStars extends BaseOreService {
+  static category = 'rating'
+
+  static route = {
+    base: 'ore/stars',
+    pattern: ':pluginId',
+  }
+
+  static examples = [
+    {
+      title: 'Ore Stars',
+      namedParams: {
+        pluginId: 'nucleus',
+      },
+      staticPreview: this.render({ stars: 1000 }),
+      documentation,
+      keywords,
+    },
+  ]
+
+  static defaultBadgeData = {
+    label: 'stars',
+  }
+
+  static render({ stars }) {
+    return {
+      message: metric(stars),
+      color: 'brightgreen',
```

Comment:
The `brightgreen` color is used on our badges to indicate some kind of rousing success, like a successful builds or a ton of downloads. In most other cases where we're simply displaying a count of stars (vs a star rating scale) we use the standard color `blue` for informational content, like the GitHub and Docker Stars badges and we should do the same here.